.github/workflows/test.yml

```diff
@@ -175,7 +190,7 @@ jobs:
   test-proxy:
     runs-on: ubuntu-latest
     container:
-      image: alpine/git:latest
+      image: fdev321123/ubuntu-with-git:latest
```

Comment:
We should be very careful about adding in external dependencies to our workflows - https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions

If this is something we internally built, could we publish it to ghcr.io within this repository or org?

Reply:
Agreed.  This was something @fhammerl provided as a proof-of-concept, but I was just reading up about how to publish packages to ghcr.io and like the idea of hosting this container image at ghcr.io/actions (see [neighboring packages here](https://github.com/orgs/actions/packages?ecosystem=container&visibility=public))